### PR TITLE
[jsk_2016_01_baxter_apc] segmentation_in_bin split into right and left launch

### PR DESCRIPTION
**Pull request by knorth55**

### CODE REVIEW

File: jsk_2016_01_baxter_apc/launch/segmentation_in_bin.launch [hunk 1]
```diff
@@ -28,46 +24,26 @@
...
-
-  <node name="sib_centroid_publisher"
-        pkg="nodelet" type="nodelet"
-        args="load jsk_pcl/CentroidPublisher $(arg MANAGER)">
-        <remap from="~input" to="/sib_extract_pi/output"/>
-  </node>
+  <include file="$(find jsk_2016_01_baxter_apc)/launch/include/left_hand_segmentation_in_bin.launch">
+    <arg name="INPUT_CLOUD" value="$(arg INPUT_CLOUD)" />
+    <arg name="MANAGER" value="$(arg MANAGER)" />
+    <arg name="JSON" value="$(arg json)" />
```
Comment (yuyu2172) [inline jsk_2016_01_baxter_apc/launch/segmentation_in_bin.launch]: Can you make left_hand_segmentation_in_bin.launch and right_hand_segmentation_in_bin.launch into one file?
I think you can by using an argument that indicates left or right.

If you think that it is too complicating, then that is fine too.

Comment (yuyu2172) [inline jsk_2016_01_baxter_apc/launch/segmentation_in_bin.launch]: fixed?
If it is, then this is merge ok.
I am going to go to sleep.

**Comment by wkentaro** [comment]
@yuyu2172 ping.

**Comment by wkentaro** [comment]
ping

**Comment by knorth55** [comment]
ping

**Comment by knorth55** [comment]
@yuyu2172 ping

**Pull request merged**